src/fdb5/remote/Connection.cc

```diff
@@ -19,20 +19,20 @@ void Connection::teardown() {
             // all done - disconnecting
             Connection::write(Message::Exit, false, 0, 0);
         } catch(...) {
-            // if connection is already down, no need to escalate 
+            // if connection is already down, no need to escalate
         }
     }
     try {
         // all done - disconnecting
         Connection::write(Message::Exit, true, 0, 0);
     } catch(...) {
-        // if connection is already down, no need to escalate 
+        // if connection is already down, no need to escalate
     }
 }
 
 //----------------------------------------------------------------------------------------------------------------------
 
-void Connection::writeUnsafe(bool control, const void* data, size_t length) {
+void Connection::writeUnsafe(const bool control, const void* data, const size_t length) const {
```

Comment:
If you make all parameters const you should also make the pointer const, i.e. `const void * const`

Reply:
there is no **should**, and feel free to contribute it as a commit!